scripts/release-notes/index.js

```diff
@@ -279,17 +279,27 @@ const formatCommits = (commits) => {
   return lines.join('\n');
 };
 
+const formatDate = () => {
+  const now = new Date();
+  return now.toISOString().split('T')[0]; // Format as YYYY-MM-DD
+};
+
 const output = ({ warnings, types }, commits) => {
   console.log(`
 ---
 title: "${MILESTONE_NAME} release notes"
 linkTitle: "${MILESTONE_NAME}"
+date: ${formatDate()}
 weight:
 description: >
```

Comment:
We actually don't want the front matter to have a new `date` field.  Instead, if you look at an [existing page](https://github.com/medic/cht-docs/blob/9ea182c123cc65d920843edf1dce345ccf339fb5/content/en/releases/4.18.0.md?plain=1#L6), you'll see it's placed in the description like this: 

```suggestion
weight:
description: >
   Released ${formatDate()}
```